Tests/UnitTests/Scripts/tests.js

```diff
@@ -211,6 +211,241 @@ describe("clearTimeout", function () {
     });
 });
 
+// URL
+describe("URL", function () {
+
+    // Currently all of the properties that the polyfill has implemented
+    function checkURL(
+        url,
+        href,
+        hostname,
+        origin,
+        pathname,
+        search) {
+        expect(url).to.have.property("hostname", hostname);
+        expect(url).to.have.property("href", href);
+        expect(url).to.have.property("origin", origin);
+        expect(url).to.have.property("pathname", pathname);
+        expect(url).to.have.property("search", search);
+    }
+
+    // Base URLs:
+    let baseUrl = 'https://developer.mozilla.org';
+
+    it('should load URL with no pathname / search', function () {
+        // Standard URL (No pathname, no search)
+        let url = new URL(baseUrl);
+        // => 'https://developer.mozilla.org'
+        checkURL(url,
+            'https://developer.mozilla.org',  // href
+            'developer.mozilla.org',          // hostname
+            'https://developer.mozilla.org',  // origin
+            '',                               // pathname
+            ''                                // search
+        );
+    });
+
+    it("should load URL with pathname (no search)", function () {
+        // Augment URL with pathname (no search)
+        url = new URL(baseUrl + '/en-US/docs');
```

Comment:
FYI: We can also use string interpolation syntax with backquotes:
```suggestion
        url = new URL(`${baseUrl}/en-US/docs`);
```